packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -107,5 +107,7 @@ export const messages = {
   force_sobjects_refresh: 'SFDX: Refresh SObject Definitions',
   force_project_create_text: 'SFDX: Create Project',
   force_project_create_open_dialog_create_label: 'Create Project',
-  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger'
+  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger',
+  no_refresh_if_currently_active:
+    'Refresh is already active.  Cancel the existing task if there is a need to restart'
```

Comment:
We use only one space after periods. We also have a "goal must precede action" style guideline, which basically means that "if xyz" should come before "then abc." Also, can we be more specific about what kind of refresh this is, or is this message likely to be used for different types of refreshes? Maybe something like this would work?

'A refresh of your sObject definitions is already underway. If you need to restart the process, cancel the running task.'

Reply:
Group this message along with the other sobject one, and make the name conform to the conventions.

Reply:
@ruthemmanuelle @chowwinston  done - please verify